packages/buckets/package.json

```diff
@@ -13,7 +14,7 @@
   },
   "scripts": {
     "build": "npx tsc -b ./tsconfig.json",
-    "clean": "npx rimraf ./dist ./tsconfig.tsbuildinfo",
+    "clean": "npx rimraf ./dist",
```

Comment:
the whole build setup is now in dist, so we can just delete that

Reply:
you _might_ want to delete these build assets before pulling these changes so you don't have these hanging around?